modules/account-lib/test/unit/mpc/curves/secp256k1.ts

```diff
@@ -0,0 +1,106 @@
+    valueTwo.should.equal(testValue);
+  });
+
+  it('should correctly perform inverts', function () {
+    // Had to convert bigint to string as comparison was not properly working in mocha
+    const testOne = sec256k1.scalarInvert(BigInt(3)).toString();
+    testOne.should.equal('77194726158210796949047323339125271901891709519383269588403442094345440996225');
+
+    const testTwo = sec256k1.scalarInvert(sec256k1.order() + BigInt(1)).toString();
+    testTwo.should.equal('1');
+
+    const testThree = sec256k1
+      .scalarInvert(BigInt('57657865876576467547584635432343132435146576543543134312435443514313234131324551034534'))
+      .toString();
+    testThree.should.equal('45199239020920791752653273479365580486664737121914751844510223387235717825519');
+
+    const testFour = sec256k1.scalarInvert(BigInt('-23232424342224')).toString();
+    testFour.should.equal('109007816100548644677753129517258866095932985532097924870123596204155595187610');
+
+    const testFive = sec256k1.scalarInvert(sec256k1.order() + testValue).toString();
+    testFive.should.equal('16664411509403755791375590925833668071820872792496863322396593062116822075060');
+  });
+
+  it('should correctly perform point add', function () {
+    const A = BigInt('0x0279be667ef9dcbbac55a06295ce870b07029bfcdb2dce28d959f2815b16f81798');
+    const B = BigInt('0x0252972572d465d016d4c501887b8df303eee3ed602c056b1eb09260dfa0da0ab2');
+    const publicKey = sec256k1.pointAdd(A, B).toString();
+    publicKey.should.equal('449973406896538015988913407864183977074571439549869550234638933112191976505998');
+  });
+
+  it('should correctly perform base point multiplication', function () {
+    const privKey = BigInt('0x79FE45D61339181238E49424E905446A35497A8ADEA8B7D5241A1E7F2C95A04D');
+    const publicKey = sec256k1.basePointMult(privKey).toString(16);
+    publicKey.should.equal('32a574ea59cae80b09d6ba415746e9b031abfbe83f149b43b37be035b87164872');
+  });
+
+  it('should correctly verify signature', function () {
+    const publicKey = BigInt('0x33ccfc7998b1ce2fad8b3f287d873f07ca8b3037897c2136c11100c02213c028e');
+    const signature = bigIntToBufferBE(
+      BigInt(
+        '0x6a43d5da66ec4b4c6eb307d18791b9744cc59dd2402ad395bb9efb513898c07f767139118b76e387564bf572b2cc481dbf3068b7ac27620f83a2819d735b7e5f',
+      ),
+    );
+    const message = bigIntToBufferBE(BigInt('0xbe5548911159dca31c02102c9df8c842adb50ac6def360c343ee5586f2749c0c'));
+    let verify = sec256k1.verify(message, signature, publicKey);
+    verify.should.equal(true);
+    const messageWrong = bigIntToBufferBE(BigInt('0xd3b91b70417e766b738288cb1c5f6ef13a607011134544d4bce2ebf6ed5944ef'));
+    verify = sec256k1.verify(messageWrong, signature, publicKey);
+    verify.should.equal(false);
+  });
+
+  it('should have the correct curve order', function () {
+    const curveOrder = sec256k1.order().toString();
+    curveOrder.should.equal('115792089237316195423570985008687907852837564279074904382605163141518161494337');
```

Comment:
Let's either specify this as a hex value, as it is defined in [documentation](https://en.bitcoin.it/wiki/Secp256k1) of secp256k1:
`n = FFFFFFFF FFFFFFFF FFFFFFFF FFFFFFFE BAAEDCE6 AF48A03B BFD25E8C D0364141`